config/filter.d/ignorecommands/apache-fakegooglebot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Based on: https://isc.sans.edu/forums/diary/When+Google+isnt+Google/15968/
```

Comment:
thinking about it, we should have made ignorecommand more generic, just requiring specification as "command <IP>" and any other options. then we could just create this generic helper "ignoreifnamematching" which would get a hostname pattern as the first argument and IP as the 2nd, i.e. ```ignoreifnamematching 'crawl-.*\.googlebot\.com' <IP>``` and be done but also scale easily for other similar cases... what do you think?